Neos.ContentRepository.Core/Classes/Feature/WorkspaceCreation/Event/RootWorkspaceWasCreated.php

```diff
@@ -29,7 +31,7 @@
  *
  * @api events are the persistence-API of the content repository
  */
-final readonly class RootWorkspaceWasCreated implements EventInterface
+final readonly class RootWorkspaceWasCreated implements EventInterface, EmbedsWorkspaceName
```

Comment:
hmm i just stubled upon the fact that we do not use `EmbedsContentStreamId` despite that it does in `newContentStreamId` im sure that is done for a reason ... and the tests provide me with reason:

> Cannot extract version number, as it was projected on wrong stream "Workspace:live", but needs to start with ContentStream:

so its only allowed to implement `EmbedsContentStreamId` to events that are also _published_ on a content stream. Good to know.